src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -112,6 +112,14 @@ private CallbackValidations() {
             : invalid("Exception case has no Id");
     }
 
+    @Nonnull
+    public static Validation<String, CaseDetails> hasId(CaseDetails theCase) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Linguistics Antipattern - The method 'hasId' indicates linguistically it returns a boolean, but it returns 'Validation'](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4577310)